### PR TITLE
fix(site): use WebGL renderer for terminal

### PR DESCRIPTION
**Pull request by ammario**
fix(site): use WebGL renderer for terminal

It's faster than Canvas.

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/coder/coder/pull/9320).
* __->__ #9320
* #9259

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -103,7 +103,9 @@
     "xterm": "5.2.1",
     "xterm-addon-canvas": "0.4.0",
```
Comment (matifali) [inline site/package.json:104]: Should canvas addon be removed or is it still being used somewhere?

**Comment by mafredri** [comment]
We should confirm that we don't re-introduce a regression with this change (c.f. https://github.com/coder/coder/issues/7953, https://github.com/coder/coder/pull/8138).

That was DOM vs canvas, though. So probably not applicable here. But better safe than sorry.

**Comment by matifali** [comment]
@mafredri I tested htop and it was working fine no overlapping or anything, even with resizing. Do you suggest any further tests. we can test in our PR deployment.

**Comment by mafredri** [comment]
@matifali nice! There's a lot we could test but I think `htop` is a good benchmark unless we know other potentially problematic software. A few others that come to mind: `vim`, `emacs`.

**Comment by matifali** [comment]
cc: @BrunoQuaresma

**Pull request merged**